app/gimletlet/app.toml

```diff
@@ -161,10 +161,10 @@ task-slots = ["sys"]
 [tasks.net]
 path = "../../task/net"
 name = "task-net"
-stacksize = 3800
+stacksize = 8192
```

Comment:
Dang, that's a lot of stack -- how close are we to using all of that, or is this padding left over from development? (i.e. what does `humility stackmargin` tell you)